Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,28 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string firstName = "Tom";
```

Comment:
These values should come from the command line by using Console.ReadLine() to wait and read user input.

Reply:
I agree, please fix.

Reply:
The code was rewritten to properly take input and give proper output.